src/PlaywrightSharp.Tests/TestConstants.cs

```diff
@@ -63,5 +63,23 @@ public static LaunchOptions GetHeadfulOptions()
             "        http://localhost:<PORT>/frames/frame.html (dos)",
             "    http://localhost:<PORT>/frames/frame.html (aframe)"
         };
+
+        internal static readonly BrowserContextOptions _iPhone6 = new BrowserContextOptions()

```

Comment:
How about removing the `_`?

Reply:
Someone's style guides kicked in and made me rename :P 

But yeah, good catch. I'll make it nicer.